### PR TITLE
add telegram-purple to the list of plugins

### PR DESCRIPTION
**Pull request by krydos**
Hi @mbologna 
Thanks for the project it was super useful for me. 

Since I use Telegram I've extended Dockerfile with telegram-purple. If you consider it useful it would be great if you'll merge it.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -16,11 +17,21 @@ make && \
 make install && \
 make install-etc && \
 make install-dev && \
+# install skypeweb
 cd && \
 tar zxvf 1.4.tar.gz && \
 cd skype4pidgin-1.4/skypeweb && \
 make && \
 make install && \
+# skypeweb end
```
Comment (mbologna) [inline Dockerfile]: please remove this line (the next line already declares a new install phase)

File: Dockerfile [hunk 2]
```diff
@@ -16,11 +17,21 @@ make && \
...
+# install telegram purple
+cd && \
+tar zxvf telegram-purple_1.3.1.orig.tar.gz && \
+cd telegram-purple && \
+./configure && \
+make && \
+make install && \
+# telegram end
```
Comment (mbologna) [inline Dockerfile]: please remove this line and add a line to delimit the next install phase ("install bitlbee facebook")

File: Dockerfile [hunk 3]
```diff
@@ -53,4 +64,3 @@ RUN touch /var/run/bitlbee.pid && \
 USER daemon
 EXPOSE 6667
 CMD ["/usr/local/sbin/bitlbee", "-c", "/usr/local/etc/bitlbee/bitlbee.conf", "-n", "-u", "daemon"]
-
```
Comment (mbologna) [inline Dockerfile]: Please leave a newline at the end of the file.

**Review by mbologna — CHANGES_REQUESTED** [review]

**Comment by mbologna** [comment]
Hi! Some comments to fix (see inline).
Please also modify the cleanup part to get rid of installed packages and extracted installation files in order to decrease container size.

As soon as these issues are addressed, this PR is good to be merged.
Thanks for the useful contribution!

**Comment by krydos** [comment]
Awesome! Thanks for the feedback. 

I'll let you know when they'll be fixed.

**Comment by krydos** [comment]
hi @mbologna 
All your comments are addressed.

**Review by mbologna — APPROVED** [review]

**Comment by mbologna** [comment]
Thanks for the useful contribution. This is ready to be merged.

**Pull request merged**